Makefile

```diff
@@ -174,6 +175,7 @@ _SKIP_ARGS := $(foreach arg,$(strip $(GINKGO_SKIP)),-ginkgo.skip="$(arg)")
 endif
 .PHONY: e2e-tests
 e2e-tests: CONTAINER_RUNTIME?=docker ## Env variable can override this default
+export CONTAINER_RUNTIME
```

Comment:
No need to export this already done here https://github.com/metal3-io/cluster-api-provider-metal3/blob/3c299790d3af391e85de6b64852c2fd5ae0a953c/scripts/environment.sh#L39-L42